Automattic/jetpack
# Protect: Add notifications

#### Changes proposed in this Pull Request:

* Adds notifications to the menu item and the admin bar

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate Protect
* Set the response_type to `complete_green`
* Confirm you don't see anything in the Protect menu item neither in the admin bar
* Set the response_type constant to `complete`
* Confirm you see notifications in the menu item and in the admin bar
* Confirm the numbers displayed match the total number of vulneratbilities
* Confirm that clicking on the admin bar takes you to the Protect page

![Captura de tela de 2022-04-27 13-06-17](https://user-images.githubusercontent.com/971483/165562629-09812f3a-dada-4b16-a39a-4ead0c75877c.png)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202064326332574
